src/components/pages/LandingPage/LandingPage.js

```diff
@@ -16,7 +16,7 @@ import FooterLanding from '../FooterLanding/FooterLanding';
 //Images acquired from unsplash.com
 
 const LandingPage = () => {
-  const { loginWithRedirect } = useAuth0();
+  const { loginWithRedirect, isAuthenticated } = useAuth0();
```

Comment:
relying on auth0 to check whether the user is authenticated to conditionally render the login component is smart